index.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+//require('../')()
+
+const findLinks = require('./utils/findLinks')
+const linksFunctions = require('./utils/validateAndStatusLinks')
+const fs = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+
+const readDirectory=(uri, comandLine4, comandLine5)=>{
+    let direc = fs.readdirSync(uri)
+    direc.forEach(elemnt => {
+       
+        let lastmd = elemnt.lastIndexOf(".");
+        let newF = elemnt.slice(lastmd);
+        if (newF === '.md') {
+            let newPath = path.join(uri, elemnt)
+            let expR = /\\/gi;
+            let realPath = newPath.replace(expR, "\/");
+            let fileDir = fs.readFileSync(realPath, 'utf-8')
+            findLinks(fileDir, realPath)
+                .then(linksReal => {
+                    return linksFunctions.getStatusLink(linksReal, comandLine4, comandLine5, realPath)
+                })
+                .catch(er => console.log(er))
+        }
+        else {
+
+            console.log(chalk.gray.bold("It is not a .md extention file ******" + "  " + elemnt))
+        }
+    })
+}
+
+
+const readFile= (uri, comandLine4, comandLine5)=>{
+    let probe = uri.lastIndexOf(".");
+    let typeFile = uri.slice(probe);
+    if (typeFile === ".md") {
+        let fileReal = fs.readFileSync(uri, 'utf-8')
+        findLinks(fileReal, uri)
+            .then(linksReal => {
+                return linksFunctions.getStatusLink(linksReal, comandLine4, comandLine5, uri)
+            })
+            .catch(er => console.log(er))
+    } else { 
+        let messErr= 'you need a file with .md extention o you missed the flag --file before your path :) '
+        console.log(chalk.redBright(messErr))
+        return messErr
+         }
+}
+
+
+let allTheLinks = () => {
```

Comment:
Seria bueno estandarizar el uso de const o let para definir las arrow functions 
```suggestion
const allTheLinks = () => {
```